Public/Src/App/Bxl/Args.cs

```diff
@@ -366,18 +366,18 @@ public bool TryParse(string[] args, PathTable pathTable, out ICommandLineConfigu
                         OptionHandlerFactory.CreateBoolOption(
                             "enableGrpc",
                             sign => distributionConfiguration.IsGrpcEnabled = sign),
+                        OptionHandlerFactory.CreateBoolOption(
+                            "enableIncrementalFrontEnd",
+                            sign => frontEndConfiguration.EnableIncrementalFrontEnd = sign),
                         OptionHandlerFactory.CreateBoolOptionWithValue(
                             "enableLazyOutputs",
                             (opt, sign) => HandleLazyOutputMaterializationOption(opt, sign, schedulingConfiguration)),
                         OptionHandlerFactory.CreateOption(
                             "engineCacheDirectory",
                             opt => layoutConfiguration.EngineCacheDirectory = CommandLineUtilities.ParsePathOption(opt, pathTable)),
                         OptionHandlerFactory.CreateBoolOption(
-                            "enableIncrementalFrontEnd",

```

Comment:
Nit - in the future I'd suggest not making cleanup type of changes in something we're going to have to cherry pick to prevent merge conflicts

Reply:
Noted.

---
In reply to: [289939300](https://github.com/microsoft/BuildXL/pull/398#discussion_r289939300) [](ancestors = 289939300)